README.md

```diff
@@ -316,6 +316,45 @@ pipenv run python gatorgrouper_cli.py --file filepath --method graph
 --preferences-weight-match PREFERENCES_WEIGHT_MATCH --objective-measures LIST
 --objective-weights LIST
 ```
+### Student Data Files
+
+Gatorgrouper can import student data from a supplied CSV file using the 
+`--file data.csv` flag. Similarly, data about student's gropuing preferences
+can be imported via `--preferences preferences.csv`.
+
+The student data file contains one line for each student, along with a
+comma-separated list of data points for that student, e.g.:
+```
+Bob,True,1.0,3.2,Python,False
```

Comment:
Can you provide a sample of the ordering for this kind of statement? For example:
```
<name>,<male?>,<avg>,<max>,<language>,<proficient>
bob,True,1.0,3.2,Python,False